### PR TITLE
S44792925 Improved UNet

### PR DESCRIPTION
**Pull request by patapan**
Student: S44792925
Task: Improved UNet

This PR includes an implementation of the improved UNet running on the ISIC 2018 Dataset with a dice coefficient above 0.8.

The structure includes
- README.md (readme file)
- images folder (results images)
- main.ipynb (driver script)
- model.py (containing Improved UNet model)

### CODE REVIEW

**Comment by patapan** [comment]
Please tell me if there are any areas of the readme that you feel could be improved upon :)

**Comment by wooboyeong** [comment]
README
* Don't plot loss, accuracy, Dice on the same plot - should be plotted separately (plot train loss and val loss on the same plot instead)

General Comments
* Code OK for EASY
* Commit messages OK
* Pull request OK

**Comment by shakes76** [comment]
## TF/Torch Usage
Adequate use and implementation
## Good Practice (Design/Commenting)
Good spacing and comments
No Header blocks -1
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present 
No Data leakage 
Difficulty: Easy -10
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch)
No feedback required
Request Description good
